### PR TITLE
[RF-959] Archive External Accounts

### PR DESCRIPTION
**Pull request by ksalmon**

### CODE REVIEW

**Comment by RockinRonE** [comment]
Plaid and PNC (my bank) do not like each other and I'm unable to connect my account, therefore I'm not sure how to test this. Any suggestions?

**Comment by ksalmon** [comment]
> Plaid and PNC (my bank) do not like each other and I'm unable to connect my account, therefore I'm not sure how to test this. Any suggestions?

This is staging, which is the sandbox Plaid env. You don't need to worry about connecting your real bank.

**Comment by ksalmon** [comment]
@RockinRonE @KramerJeff

Testing Strategy:
Option #1) You will need to find a user that has had an already existing external for 30 days and archive that account and add a new one.

Option #1) Create a new user and attach an account, and archive that account, and then work with QA to bypass the 30 day validations.

**Comment by ksalmon** [comment]
@RockinRonE @KramerJeff 
Testing Video: https://www.loom.com/share/73fe5a43836b4c9599da9d1e68243f83

File: screens/ExternalAccounts/ExternalAccountScreen.tsx [hunk 1]
```diff
@@ -148,20 +161,19 @@ const ExternalAccountScreen = (): JSX.Element => {
       return (
         <>
           <Heading5 textAlign="center" style={styles.heading}>
-            Select an Account
+            Select account from the avaliable accounts:
```
Comment (KramerJeff) [inline screens/ExternalAccounts/ExternalAccountScreen.tsx:164]: Typo: should be 'available'

**Review by KramerJeff — APPROVED** [review]
Approved with a small typo. This endpoint is relatively simple so I can't think of any sort of improvements on this PR.

**Pull request merged**